XamlStyler.Core/Options/StylerOptions.cs

```diff
@@ -271,6 +271,15 @@ public StylerOptions(string config = "")
         public int CommentSpaces { get; set; }
 
         // Configuration
+
+        [Category("XAML Styler Configuration")]
+        [RefreshProperties(RefreshProperties.All)]
+        [DisplayName("Search to drives root")]
+        [Description("When set to true, XAML Styler will look for an external XAML Styler configuration file not only up through your solution directory, but up through the drives root of the current solution so you can share one configuration file through multiple solutions.\r\n\r\nDefault Value: true")]
+        [DefaultValue(true)]
```

Comment:
IMO, this should be false by default, so we do not inadvertently cause back compat issues.

Reply:
I hope I've done this the right way: The last commit should include this change...